packages/api-explorer/src/test-data/sdkLanguages.ts

```diff
@@ -0,0 +1,36 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2022 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+export const sdkLanguageMapping = {
+  py: 'Python',
+  ts: 'TypeScript',
+  cs: 'C#',
+  kt: 'Kotlin',
+  swift: 'Swift',
+  go: 'Go',
+  all: 'All',
+}
```

Comment:
note that this was purposely hardcoded for testing purposes. Supporting a new language in API Explorer requires a minor manual effort since we have dependencies such as `prism` where a human needs to confirm correctness.

Reply:
This needs to be created from iterating through the existing language definitions in sdk-codegen before this PR can be merged. Let's not have a different hard-coded list of languages.